internal/controllers/machineset/machineset_controller.go

```diff
@@ -424,6 +424,7 @@ func (r *Reconciler) syncMachines(ctx context.Context, machineSet *clusterv1.Mac
 			}
 
 			// Set all other in-place mutable fields that impact the ability to tear down existing machines.
+			m.Spec.ReadinessGates = machineSet.Spec.Template.Spec.ReadinessGates
```

Comment:
I first thougt we also have to add it to computeDesiredMachine, but seems like it works because we copy the entire template there. Can we please make this more explicit and also add it in machineset_controller.go l.732?

Reply:
done